.github/workflows/main.yml

```diff
@@ -38,11 +38,11 @@ jobs:
 
       - name: Lint
         run: |
-          "${GITHUB_WORKSPACE}/bin/bazel" run //:check
+          ./lint.sh
```

Comment:
@lucapette Since part of the lint is tests and part is a command we cannot make this a single Bazel command anymore.